app/views/articles/_full_comment_area.html.erb

```diff
@@ -2,11 +2,11 @@
   <section id="comments" data-updated-at="<%= Time.current %>" class="text-padding mb-4 border-t-1 border-0 border-solid border-base-10">
     <% if @article.show_comments %>
       <header class="relative flex justify-between items-center mb-6">
-        <h2 class="crayons-subtitle-1">Discussion</h2>
+        <h2 class="crayons-subtitle-1">Discussion <span class="comments-count" data-comments-count="<%= @article.comments_count %>">(<%= @article.comments_count %>)</span> </h2>
```

Comment:
One tiny suggestion: can we use `js-comments-count` class name here instead of `comments-count` - it would be an indicator that this class is used to reference an element in JS.

Reply:
Sure, Will make the suggested change and update PR.